bird-slideshow.py

```diff
@@ -17,7 +17,7 @@
 from bs4 import BeautifulSoup
 
 class Config:
-    def __init__(self, config_file: str | None) -> None:
```

Comment:
Do you know what version of python supports this syntax?

Reply:
Vertical Bar was introduced for Union types in 3.10